source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -5014,11 +5014,11 @@ function raiseImageLoadedEvent(image: Image) {
     }
 }
 
-function raiseAnchorClickedEvent(element: CardElement, anchor: HTMLAnchorElement): boolean {
-    let card = element.getRootElement() as AdaptiveCard;
+function raiseAnchorClickedEvent(textBlock: TextBlock, anchor: HTMLAnchorElement): boolean {
+    let card = textBlock.getRootElement() as AdaptiveCard;
```

Comment:
There is no need to change this. Please keep it as CardElement. We may support inline actions in elements that are not TextBlock in the future.